### ISSUE TITLE
Can you provide a evaluation file which can reproduce your results on YouTube-VIS validation set?

### ISSUE THREAD

**Issue by vealocia** [sabarim/STEm-Seg#7]
Thanks for your great works, can you provide a result file to reproduce your results on YouTube-VIS dataset?

**Comment by luoww1992** [comment]
> 感谢您的出色工作，您能否提供结果文件以在YouTube-VIS数据集上重现您的结果？
can you run it in YouTube-VIS ?
can you share how to make training dataset in YouTube-VIS ?  i don't know how to make the value: segmentations in youtube_vis_train.json from annotation['segmentations']  by pycocotool?
